ansible_files/roles/equinix/heterogeneous_create_infra/tasks/wait_for_instance.yaml

```
@@ -0,0 +1,19 @@
+    - name: Download Metal cli
+      ansible.builtin.get_url:
+        url: "https://github.com/equinix/metal-cli/releases/download/v{{ metal_version }}/metal-linux-amd64"
+        dest: /tmp/metal
+        mode: '0775'
+        checksum: "sha256:https://github.com/equinix/metal-cli/releases/download/v{{ metal_version }}/metal-cli_{{ metal_version }}_checksums.txt"
+    
+    - name: "Wait for ARM instance {{ item }}"
```

```suggestion
    - name: "Wait for plans to be available"
```
`{{ item }}` doesn't show properly:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_assisted-test-infra/2313/pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-day2-arm-workers/1727238404069920768/artifacts/e2e-metal-assisted-day2-arm-workers/assisted-equinix-setup/build-log.txt